packages/autocomplete-core/src/utils/__tests__/createCancelablePromise.test.ts

```diff
@@ -96,82 +96,13 @@ describe('createCancelablePromise', () => {
 
     await runAllMicroTasks();
 
-    expect(onFulfilled).not.toHaveBeenCalled();
-    expect(onRejected).not.toHaveBeenCalled();
-    expect(onFinally).not.toHaveBeenCalled();
-  });
-
-  test('triggers `finally` handler with `runWhenCanceled=true` when the cancelable promise is canceled then resolves', async () => {
```

Comment:
don't the "true" tests still need to exist?

Reply:
Yep it's [still here](https://github.com/algolia/autocomplete/blob/d68de50779d5f39bb645d5da092ad3ab1c7fc471/packages/autocomplete-core/src/utils/__tests__/createCancelablePromise.test.ts#L81-L127), but now it's default behavior.